.github/workflows/azure-static-web-apps-victorious-sand-0da50520f.yml

```diff
@@ -1,45 +0,0 @@
-name: Azure Static Web Apps CI/CD

```

Comment:
Did we intend to remove GH workflow with this PR?

Reply:
Yes as is it a dupe with the `preview-branches.yml` file. Sorry for shoehorning it in – I can pull it out into another PR if you prefer.